modules/tinymce/CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The sidebar element now has accessibility role `region` when visible and `presentation` when hidden. #TINY-9517
 
 ### Fixed
+- It was not possible to open and close tree component nodes using left and right arrow keys. #TINY-9640
```

Comment:
We don't need this entry, since the tree component is going to be released. There is the appropriate entry in Added section for it.

Reply:
Done!